src/ACadSharp/Header/CadHeader.cs

```diff
@@ -1194,6 +1195,78 @@ public string CurrentDimensionStyleName
 			}
 		}
 
+		public MultiLeaderStyle CurrentMultiLeaderStyle
+		{
+			get
+			{
+				if (this.Document == null) {
+					return this._currentMultiLeaderStyle;
+				}
+				else {
+					return this.Document.MLeaderStyles[this.CurrentMultiLeaderStyleName];
+				}
+			}
+			private set {
+				this._currentMultiLeaderStyle = value;
+			}
+		}
+
+		public string CurrentMultiLeaderStyleName
+		{
+			get
+			{
+				if (this.Document == null)
+				{
+					return this._currentMultiLeaderStyle.Name;
+				}
+				else
+				{
+					DictionaryVariable variableDictionaryEntry = ensureVariableDictionaryEntryExists(this.Document);
+					return variableDictionaryEntry.Value;
+				}
+			}
+			set
+			{
+				if (string.IsNullOrEmpty(value))
+				{
+					throw new ArgumentNullException("value");
+				}
+				if (this.Document != null)
+				{
+					this._currentMultiLeaderStyle = this.Document.MLeaderStyles[value];
+
+					//  There is no accessible system variable $CMLEADERSTYLE
+					//  The current MultiLeaderStyle name is held in the Variable Dictionary
+					//	create or update this entry
+					DictionaryVariable variableDictionaryEntry = ensureVariableDictionaryEntryExists(this.Document);
+					variableDictionaryEntry.Value = value;
+				}
+				else
+				{
+					this._currentMultiLeaderStyle = new MultiLeaderStyle(value);
+				}
+			}
+		}
+
+		private static DictionaryVariable ensureVariableDictionaryEntryExists(CadDocument doc) {
```

Comment:
To make this method more generic and be able to use for the future variables in the dictionary we should parametrize to make it more generic, here is my proposal:

```C#
private static DictionaryVariable ensureVariableDictionaryEntryExists(CadDocument doc, string name, string value)
```